### PR TITLE
Update user-tags.md - Permissions update & Quarantine integration

### PR DESCRIPTION
**Pull request by melanie-cohen**
Updating permissions for the Priority account tag and adding Quarantine integration.

### CODE REVIEW

**Comment by melanie-cohen** [comment]
@chrisda and @faoquong, updating documentation.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/user-tags.md [hunk 1]
```diff
@@ -56,8 +57,9 @@ To see how user tags are part of the strategy to help protect high-impact user a
...
 
 - You need to be assigned permissions in the Microsoft 365 Defender portal before you can do the procedures in this article:
-  - To create, modify, and delete user tags, you need to be a member of the **Organization Management** or **Security Administrator** role groups.
-  - To add and remove members from existing user tags, you need to be a member of the **Organization Management**, **Security Administrator**, or **Security Operator** role groups
+  - To create, modify, and delete custom user tags, you need to be a member of the **Organization Management** or **Security Administrator** role groups.
+  - To add and remove members from the Priority Account system tag, you need to be a member of the **Security Administrator** and **Exchange Admin** role groups
+  - To add and remove members from existing custom user tags, you need to be a member of the **Organization Management**, **Security Administrator**, or **Security Operator** role groups
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/user-tags.md]:
```suggestion
  - To add and remove members from the Priority Account system tag, you need to be a member of the **Security Administrator** and **Exchange Admin** role groups.
  - To add and remove members from existing custom user tags, you need to be a member of the **Organization Management**, **Security Administrator**, or **Security Operator** role groups.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggested changes, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**